maxim/logger/fireworks/client.py

```
@@ -0,0 +1,200 @@
+import functools
+from ..logger import Logger, Generation, Trace
+from ...scribe import scribe
+from typing import Any, Optional
+from fireworks import LLM
```

Ruff (0.11.9)

`fireworks.LLM` imported but unused

Remove unused import: `fireworks.LLM`

(F401)